CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-# dbt_unified_rag v0.1.0
+# dbt_unified_rag v0.1.0-a2
+
+## Bug Fixes
+- For Snowflake destinations, we have removed the post-hook from the `rag__unified_document` which generated the `rag__unified_search` Cortex Search Service. 
+    - While the Search Service worked when deployed locally, there were issues identified when deploying and running via Fivetran Quickstart. In order to ensure Snowflake users are still able to take advantage of the `rag__unified_document` end model, we have removed the Search Service from execution until we are able to verify it works as expected on all supported orchestration methods.
```

Comment:
Is there a way for them to run it themselves?

Reply:
Great point! I just added some additional documentation to share how they can run it themselves.